src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/NodesDnApiAction.java

```diff
@@ -26,6 +26,7 @@
 import com.amazon.opendistroforelasticsearch.security.securityconf.impl.NodesDn;
 import com.amazon.opendistroforelasticsearch.security.ssl.transport.PrincipalExtractor;
 import com.amazon.opendistroforelasticsearch.security.support.ConfigConstants;
+import com.amazon.opendistroforelasticsearch.security.tools.OpenSearchSecurityAdmin;
```

Comment:
The @link com.amazon.opendistroforelasticsearch.security.tools.OpenDistroSecurityAdmin is currently not working without importing the class.